### PR TITLE
feat: collections intro

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Collections intro about the feature.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1969 #done

### CODE REVIEW

File: packages/shared/src/components/post/widgets/CollectionsIntro.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export function CollectionsIntro(): ReactElement {
+  const { checkHasCompleted, completeAction } = useActions();
+
+  if (checkHasCompleted(ActionType.CollectionsIntro)) {
+    return null;
+  }
+
+  return (
+    <span className="flex relative flex-row p-3 pr-2 w-full rounded-10 border border-theme-color-cabbage">
```
Comment (rebelchris) [inline packages/shared/src/components/post/widgets/CollectionsIntro.tsx:15]: The rounded-10 we always use like this? Cant recall

**Comment by capJavert** [comment]
@sshanzel we have epic branch `WT-1939-collections` so if you can point to that.

**Comment by sshanzel** [comment]
> @sshanzel we have epic branch WT-1939-collections so if you can point to that.

I was receiving some build errors due to our cards list requiring every enum item to have an implementation. I can do that but I will have to add null values to those variables for which already exists on where this is pointing.

**Comment by capJavert** [comment]
@sshanzel ok, I will merge it when that one is done. 👌

**Comment by rebelchris** [comment]
@sshanzel Think you can rebase and get access to all the missing parts now.

File: packages/shared/src/components/Pill.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ const Pill = ({
     <div
       className={classNames(
         pillSizeToClassName[size],
-        'rounded-xl',
+        'rounded-12',
```
Comment (rebelchris) [inline packages/shared/src/components/Pill.tsx:37]: This should be 10px according to design which is `rounded-10` I think I asked dragos before if we had a named variant, but see the numbers are actually our overwrites.

File: packages/shared/src/components/post/PostWidgets.tsx [hunk 3]
```diff
@@ -32,6 +34,7 @@ export function PostWidgets({
         className="hidden tablet:flex pt-6"
         contextMenuId="post-widgets-context"
       />
+      {post.type === PostType.Collection && <CollectionsIntro />}
```
Comment (rebelchris) [inline packages/shared/src/components/post/PostWidgets.tsx:37]: Is there anyway we could inject this only in the collection modal for instance?

File: packages/shared/src/components/post/widgets/CollectionsIntro.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      <div className="flex flex-col flex-1 typo-callout">
```
Comment (rebelchris) [inline packages/shared/src/components/post/widgets/CollectionsIntro.tsx:16]: I think it's subhead, but unsure as the names seem mixed.
Asking [here](https://dailydotdev.slack.com/archives/C0381BQUEN9/p1701157996730849).

**Review by rebelchris — CHANGES_REQUESTED** [review]
Almost there, some minor things :)

File: packages/shared/src/components/Pill.tsx (hunk 2 above)
Comment (sshanzel) [reply packages/shared/src/components/Pill.tsx:37]: I see, in this case, the original value was incorrect being `xl` (12px). Let me fix this.

File: packages/shared/src/components/post/PostWidgets.tsx (hunk 3 above)
Comment (sshanzel) [reply packages/shared/src/components/post/PostWidgets.tsx:37]: We don't have yet the modal/page for rendering collections. We can probably add it if we get a separate implementation of the PostContent for Collections.

Comment (rebelchris) [reply packages/shared/src/components/post/PostWidgets.tsx:37]: Cool let's leave it for now, maybe can be a end refactor once we have that :)

File: packages/shared/src/components/post/widgets/CollectionsIntro.tsx (hunk 4 above)
Comment (rebelchris) [reply packages/shared/src/components/post/widgets/CollectionsIntro.tsx:16]: @sshanzel Please note new comment in the thread

**Review by rebelchris — APPROVED** [review]

File: packages/shared/src/components/Pill.tsx (hunk 2 above)
Comment (sshanzel) [reply packages/shared/src/components/Pill.tsx]: Updated the value to 10 now 🥳

**Pull request merged**